build_library/qemu_template.sh

```diff
@@ -203,6 +208,15 @@ if [ -z "${CONFIG_IMAGE}" ]; then
     fi
 fi
 
+# Process port forwards
+QEMU_PORT_FORWARDS=""
+for port in $PORT_FORWARDS; do
+    host_port=$(echo "$port" | cut -d':' -f1)
+    guest_port=$(echo "$port" | cut -d':' -f2)
```

Comment:
Not using shell built-ins makes me cry. :stuck_out_tongue: 

```suggestion
for port in ${PORT_FORWARDS}; do
    host_port=${port%:*}
    guest_port=${port#*:}
```

Reply:
Done